specs/Economics/README.md

```diff
@@ -111,23 +110,61 @@ NEAR validators provide their resources in exchange for a reward `epochReward[t]
 
 | Name | Description |
 | - | - |
-| `proposals` | The array of all existing validators, minus the ones which were online less than `ONLINE_THRESHOLD`, plus new validators |
-| `INCLUSION_FEE` | The arbitrary transaction fee that new validators offer to be included in the `proposals`, to mitigate censorship risks by existing validators |
-| `ONLINE_THRESHOLD` | `0.9` |
+| `proposals: Proposal[]` | The array of all new staking transactions that have happened during the epoch |
+| `prev_validators` | The array of all existing validators during the epoch |
 | `epoch[T]` | The epoch when validator[v] is selected from the `proposals` auction array |
```

Comment:
There is also the kickout set with slashed validators in the last (2 or 3?) epochs, unstakers and offline, and special tomfoolery when the union of last 2 kickout sets would be everybody and we try not to kickout one (can we assume this case never happens?)

Reply:
should probably specify this, but let's do this in separate PR. You also have some of this details in EpochManager? Question still stands how these two should be unified.